scripts/init.js

```diff
@@ -1,47 +1,44 @@
 #!/usr/bin/env node
 
-const pkg = require('../package.json');
-const commander = require('commander');
 const chalk = require('chalk');
 const fs = require('fs-extra');
 const path = require('path');
 const emptyDir = require('empty-dir');
 const validatePackageName = require('validate-npm-package-name');
 const kopy = require('kopy');
 const dedent = require('dedent');
-
 const detectYarn = require('../lib/detectYarn');
 const install = require('../lib/install');
 
-let projectName;
+const args = require('../config/args');
 
-const program = new commander.Command(pkg.name)
-  .version(pkg.version)
-  .arguments('<project-directory>')
-  .usage(`${chalk.green('<project-directory>')} [options]`)
-  .action(name => {
-    projectName = name;
-  })
-  .option('--verbose', 'print additional logs')
-  .parse(process.argv);
+const projectName = args.argv[0];
 
-if (typeof projectName === 'undefined') {
-  console.error('Please specify the project directory:');
+const usage = () => {
+  console.error('Usage:');
```

Comment:
This probably shouldn't be a `console.error` call, since we display the usage guide in valid cases like `sku init --help`.